lib/cisco_node_utils/cmd_ref/vni.yaml

```diff
@@ -61,6 +61,24 @@ mapped_vlan:
     config_set: ['vlan <vlan>', '<state> vn-segment <vni> ; end']
     default_value: ''
 
+mt_full_support:
+  # This is only used for determining support for Multi-Tenancy Full
+  kind: boolean
+  /N7/:
+    default_only: true
+  else:
+    # this feature is always off on these platforms and cannot be changed
+    default_only: false
+
+mt_lite_support:
```

Comment:
My n56128P switch does not seem to support it:

n8k-136(config)# feature nv overlay
Error: feature nv overlay failed dependencies not satisfied yet

n8k-136(config)# feature vn-segment-vlan-based 
Error: vnseg_vlan control, vn-segment depends on feature-set fabricpath or feature nv overlay (0x411000bd)

If this is a license issue, how do I get around it? ie. how to check the current license; and how to fix it?